js/apps/admin-ui/src/realm-settings/user-profile/AttributesTab.tsx

```diff
@@ -86,9 +86,15 @@ export const AttributesTab = ({ setTableData }: AttributesTabProps) => {
       if (!config?.attributes) return;
 
       const translationsToDelete = config.attributes.find(
-        (attribute) => attribute.name === attributeToDelete,
+        (attribute: UserProfileAttribute) =>
+          attribute.name === attributeToDelete,
```

Comment:
This type is already inferrable, let's not add any additional type information if it's not needed.
```suggestion
        (attribute) => attribute.name === attributeToDelete,
```